docs/docs/thinking-in-react.md

```diff
@@ -93,8 +93,8 @@ Think of all of the pieces of data in our example application. We have:
 Let's go through each one and figure out which one is state. Simply ask three questions about each piece of data:
 
   1. Is it passed in from a parent via props? If so, it probably isn't state.
-  2. Does it change over time? If not, it probably isn't state.
-  3. Can you compute it based on any other state or props in your component? If so, it's not state.
+  2. Does it remain the same over time? If so, it probably isn't state.
```

Comment:
What about "does it stay constant over time"?  The "stay" helps bringing home the idea I think.

At any rate, the "no-negatives" wasn't dogmatic. "Does it remain unchanged over time" is also fine for me. Let me know which one you prefer and I'll edit accordingly.